src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -205,6 +211,13 @@ extension ExposureNotificationSettingViewController {
 					cell.configure(for: enState, delegate: self)
 					lastActionCell = cell
 				}
+			case .euTracingCell:
+				let euTracingCell = tableView.dequeueReusableCell(withIdentifier: ENSettingModel.Content.euTracingCell.cellType.rawValue, for: indexPath)
+				if let euTracingCell = euTracingCell as? EuTracingTableViewCell {
+					euTracingCell.configure(for: store.euTracingSettings!)
+					euTracingCell.selectionStyle = .none
```

Comment:
should be part of the cell / configuration and not here

Reply:
Thanks. Moved .selectionStyle to configuration